include/spirv/unified1/spirv.hpp11

```diff
@@ -3025,1075 +3025,1075 @@ inline void HasResultAndType(Op opcode, bool *hasResult, bool *hasResultType) {
-    case StorageClassRayPayloadKHR: return "RayPayloadKHR";
-    case StorageClassHitAttributeKHR: return "HitAttributeKHR";
-    case StorageClassIncomingRayPayloadKHR: return "IncomingRayPayloadKHR";
-    case StorageClassShaderRecordBufferKHR: return "ShaderRecordBufferKHR";
-    case StorageClassPhysicalStorageBuffer: return "PhysicalStorageBuffer";
-    case StorageClassHitObjectAttributeNV: return "HitObjectAttributeNV";
-    case StorageClassTaskPayloadWorkgroupEXT: return "TaskPayloadWorkgroupEXT";
-    case StorageClassCodeSectionINTEL: return "CodeSectionINTEL";
-    case StorageClassDeviceOnlyINTEL: return "DeviceOnlyINTEL";
-    case StorageClassHostOnlyINTEL: return "HostOnlyINTEL";
+    case StorageClass::UniformConstant: return "UniformConstant";
+    case StorageClass::Input: return "Input";
+    case StorageClass::Uniform: return "Uniform";
+    case StorageClass::Output: return "Output";
+    case StorageClass::Workgroup: return "Workgroup";
+    case StorageClass::CrossWorkgroup: return "CrossWorkgroup";
+    case StorageClass::Private: return "Private";
+    case StorageClass::Function: return "Function";
+    case StorageClass::Generic: return "Generic";
+    case StorageClass::PushConstant: return "PushConstant";
+    case StorageClass::AtomicCounter: return "AtomicCounter";
+    case StorageClass::Image: return "Image";
+    case StorageClass::StorageBuffer: return "StorageBuffer";
+    case StorageClass::TileImageEXT: return "TileImageEXT";
+    case StorageClass::NodePayloadAMDX: return "NodePayloadAMDX";
+    case StorageClass::CallableDataKHR: return "CallableDataKHR";
+    case StorageClass::IncomingCallableDataKHR: return "IncomingCallableDataKHR";
+    case StorageClass::RayPayloadKHR: return "RayPayloadKHR";
+    case StorageClass::HitAttributeKHR: return "HitAttributeKHR";
+    case StorageClass::IncomingRayPayloadKHR: return "IncomingRayPayloadKHR";
+    case StorageClass::ShaderRecordBufferKHR: return "ShaderRecordBufferKHR";
+    case StorageClass::PhysicalStorageBuffer: return "PhysicalStorageBuffer";
+    case StorageClass::HitObjectAttributeNV: return "HitObjectAttributeNV";
+    case StorageClass::TaskPayloadWorkgroupEXT: return "TaskPayloadWorkgroupEXT";
+    case StorageClass::CodeSectionINTEL: return "CodeSectionINTEL";
+    case StorageClass::DeviceOnlyINTEL: return "DeviceOnlyINTEL";
+    case StorageClass::HostOnlyINTEL: return "HostOnlyINTEL";
     default: return "Unknown";
     }
 }
 
 inline const char* DimToString(Dim value) {
     switch (value) {
-    case Dim1D: return "1D";
-    case Dim2D: return "2D";
-    case Dim3D: return "3D";
-    case DimCube: return "Cube";
-    case DimRect: return "Rect";
-    case DimBuffer: return "Buffer";
-    case DimSubpassData: return "SubpassData";
-    case DimTileImageDataEXT: return "TileImageDataEXT";
+    case Dim::1D: return "1D";
+    case Dim::2D: return "2D";
+    case Dim::3D: return "3D";
```

Comment:
The names of these enums should be e.g. `Dim::Dim1D` instead of `Dim::1D`, to avoid identifiers beginning with numbers.

Reply:
as you said, we know the test are working. I fixed it and tested locally, so should be good now